NetCasbin/InternalEnforcer.cs

```diff
@@ -17,34 +17,29 @@ public class InternalEnforcer : CoreEnforcer
         /// <returns></returns>
         protected bool AddPolicy(string sec, string ptype, List<string> rule)
         {
-            var ruleAdded = model.AddPolicy(sec, ptype, rule);
-            if (!ruleAdded)
+            if (model.HasPolicy(sec, ptype, rule))
             {
-                return false;
-            } 
-
+                return true;
```

Comment:
See the docs: https://casbin.org/docs/en/management-api#addpolicy

`
If the rule already exists, the function returns false and the rule will not be added.
`